hash_practice/exercises.py

```diff
@@ -2,18 +2,40 @@
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n)
+        Space Complexity: O(n)
     """

```

Comment:
👍 Just note, if the words are limited in length you have the time/space complexity right.  If not then the time complexity is O(n * m log m) where m is the length of a word.